_rules/audio-as-media-alternative-afb423.md

```diff
@@ -35,7 +35,7 @@ The auditory information of each test target is available as text (directly or v
 
 ## Expectation 2
 
-Each target element is labeled as an audio alternative for text on the page. This label is [visible][] and [included in the accessibility tree][].
+Each target element is labeled as a video alternative for text on the page by a [visible][] identifier that is [included in the accessibility tree][].
```

Comment:
I'm not a huge fan of breaking the conjuction, it makes it a bit less clear that both conditions are needed.
Maybe:
```suggestion
Each target element is labeled as a video alternative for text on the page by an identifier that is [visible][] and [included in the accessibility tree][].
```
(on all 3 rules)